cli/azd/pkg/project/service_target.go

```diff
@@ -94,6 +94,19 @@ type ServiceTarget interface {
 	) ([]string, error)
 }
 
+// GetIgnoreFileNameByKind returns the appropriate ignore file name (e.g., .funcignore, .webappignore)
+// based on the service target kind.
+func GetIgnoreFileNameByKind(kind ServiceTargetKind) string {
+	switch kind {
+	case AzureFunctionTarget:
+		return ".funcignore"
+	case AppServiceTarget:
+		return ".webappignore"
+	default:
+		return ""
+	}
+}
```

Comment:
I debated on whether or not we should have a helper method like this, or add this to the interface, but since only a few of the service targets will have this capability I figured we should add it as a helper method to not mess with the interface of the service target. But I could be convinced the other direction, depending on how you want to design this.

Reply:
I like flexibility -- simple functions are great for that purpose. I'm okay here.